contracts/v3/Onboarding.sol

```diff
@@ -0,0 +1,62 @@
+pragma solidity ^0.7.0;
+
+// SPDX-License-Identifier: MIT
+
+import './ModuleRegistry.sol';
+import './Proposal.sol';
+import './Voting.sol';
+import '../SafeMath.sol';
+
+contract OnboardingContract {
+    using SafeMath for uint256;
+
+    struct ProposalDetails {
+        uint256 amount;
+        uint256 sharesRequested;
+        bool processed;
+    }
+
+    ModuleRegistry dao;
+    uint256 public CHUNK_SIZE;
```

Comment:
`CHUNK_SIZE` is the guild size?

Reply:
no chunk size is how much shares you get for each chunk.

in the LAO, you can get shares by chunks of 1% (100,000 shares) so we need to know how many shares you  get for each chunk and how much is a chunk. makes sense?